### ISSUE TITLE
no dependency information available for 2.6.0

### ISSUE THREAD

**Issue by rkhaja** [rvesse/airline#93]
Looks like something might be broken in Maven Central, but I am not 100% positive.

I am compiling my project with the following dependency in my pom.xml:
```
        <dependency>
            <groupId>com.github.rvesse</groupId>
            <artifactId>airline</artifactId>
            <version>2.6.0</version>
        </dependency>
```
During compilation, I see the following warnings:
```
[WARNING] The POM for com.github.rvesse.airline:airline:jar:2.6.0 is missing, no dependency information available
[WARNING] The POM for com.github.rvesse.airline:airline-examples:jar:2.6.0 is missing, no dependency information available
[WARNING] The POM for com.github.rvesse.airline:airline-help-bash:jar:2.6.0 is missing, no dependency information available
[WARNING] The POM for com.github.rvesse.airline:airline-help-html:jar:2.6.0 is missing, no dependency information available
[WARNING] The POM for com.github.rvesse.airline:airline-help-man:jar:2.6.0 is missing, no dependency information available
[WARNING] The POM for com.github.rvesse.airline:airline-help-markdown:jar:2.6.0 is missing, no dependency information available
[WARNING] The POM for com.github.rvesse.airline:airline-io:jar:2.6.0 is missing, no dependency information available
```
which results in **build failure**.

I am able to successfully compile with version **2.5.0**, so this is the reason that I feel that something is wrong with **2.6.0**. Please take a look.

**Comment by rkhaja** [comment]
Hello Rob,

I wasn't able to find a `mvn help:effective-profile` goal.
Did you mean `mvn help:effective-pom` or? or some other goal? 

> Does your build/environment override the default repositories in any way?
No.

Here is the `<repositories>` section of my `mvn help:effective-pom`:
```
  <repositories>
    <repository>
      <snapshots>
        <enabled>false</enabled>
      </snapshots>
      <id>central</id>
      <name>Central Repository</name>
      <url>https://repo.maven.apache.org/maven2</url>
    </repository>
  </repositories>
```

When I build using **2.5.0** I have no problems.
When I build using **2.6.0** I see **build failure**.

**Comment by rkhaja** [comment]
I deleted `~/.m2/repository` which forced downloading of all my dependencies.  Now I'm able to compile successfully with 2.6.0.
Closing.

**Comment by rvesse** [comment]
@rkhaja My bad, should have been `mvn help:effective-pom`

You might also have been able to add a `-U` to your `mvn` invocation to force updates of missing repository metadata

**Comment by rvesse** [comment]
The root cause was probably #88 - you probably once resolved the dependency when the artifacts were missing from Maven central and your local Maven cached that fact and since they were releases did not automatically try and refresh the metadata

**Comment by rkhaja** [comment]
I did try cleaning and updating with mvn, probably using a -U flag, but don't remember exactly, though it didn't help.

Yes, the root cause was probably #88, I may have tried 2.6.0 before the fix for #88, then reverted to 2.5.0.  More recently when I tried compiling my project with 2.6.0, it must have used the cached artifact, and caused the build failure.

Thank you for investigating!
